mocks/groundPlacesDiffFileWithNothing.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
really need to put this in a mock file ?

Reply:
I don't think so, I'll use this array inside the method calling directly